spec/components/grid-spec.js

```diff
@@ -4,6 +4,7 @@ import createComponent from '../helpers/createComponent.js';
 import Grid from '../../src/renderer/components/grid.jsx';
 import generatorsList from '../fixtures/generators-list.js';
 
+
```

Comment:
If we can catch single line breaks here with eslint, I'm happy do go with double line breaks. I will investigate and get back to you